pkg/core/localize/locales/en/cmd/kafka.en.toml

```diff
@@ -319,6 +319,22 @@ one = 'Kafka instance "{{.Name}}" already exists'
 [kafka.create.error.oneinstance]
 one = 'unable to create new Kafka instance. Only one instance is allowed with an eval account'
 
+[kafka.create.region.error.invalidRegion]
+one = '''
+the region "{{.Region}}" is not available for the cloud provider "{{.Provider}}".
+Choose from: {{.Regions}}
```

Comment:
<img width="721" alt="Screenshot 2022-01-20 at 17 07 30" src="https://user-images.githubusercontent.com/11743717/150387458-4895bc74-5dfb-4f00-b6e5-19ddeb281b1e.png">

Commas should always have space after them.